assets/_scss/vendor/font-awesome/_icons.scss

```diff
@@ -378,6 +378,7 @@
 .#{$fa-css-prefix}-youtube-play:before { content: $fa-var-youtube-play; }
 .#{$fa-css-prefix}-dropbox:before { content: $fa-var-dropbox; }
 .#{$fa-css-prefix}-stack-overflow:before { content: $fa-var-stack-overflow; }
+.#{$fa-css-prefix}-speaker-deck:before { content: $fa-var-external-link; }
```

Comment:
Ahh of course.  I should've realised that was in  a "vendored" library.  I'll revisit the way I'm doing this without hacking it in.